packages/contracts-bedrock/contracts/L1/L2OutputOracle.sol

```diff
@@ -136,10 +132,9 @@ contract L2OutputOracle is OwnableUpgradeable, Semver {
             "L2OutputOracle: starting L2 timestamp must be less than current time"
         );
 
+        startingTimestamp = _startingTimestamp;
         startingBlockNumber = _startingBlockNumber;
         latestBlockNumber = _startingBlockNumber;
-        startingTimestamp = _startingTimestamp;
-        l2Outputs[startingBlockNumber] = Types.OutputProposal(_startingL2Output, block.timestamp);
```

Comment:
Note that now `getL2Output(startingBlockNumber)` will fail which is fine because there's no proposal for that block number.